feat(#108): support custom register provider (#111)
quarkiverse/quarkus-openapi-generator: README.md

## Custom Register Providers for generated api

In some cases, we need custom `RegisterProvider` for generated api, e.g. logging. You can define your own Providers in `application.properties` :

| Property Key                                                                                                                                                                        | Example                                                                                                                                |
|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
| `quarkus.openapi-generator.codegen.spec.[filename].custom-register-providers` | `quarkus.openapi-generator.codegen.spec.simple_openapi_json.custom-register-providers=org.test.Foo,org.test.Bar`<br/>Provider classes are separated by commas |

> With the above configuration, your Rest Clients will be created with a code similar to the following:

Reviewer: ```suggestion
With the above configuration, the extension generates your Rest Clients with a code similar to the following:
```
